File: App.Client/src/api/services/table/models/EditTableConfigInput.ts
/**
 * Fast.Center.Service.Table.Dto.EditTableConfigInput 编辑表格配置输入
 */
export interface EditTableConfigInput {
  /**
   * 表格Id
   */
  tableId?: number;
  /**
   * 表格名称
   */
  tableName?: string;
  /**
   * 备注
   */
  remark?: string;
  /**
   * 更新版本控制字段
   */
  rowVersion?: number;
}

File: Api.Server/src/Services/Center/Table/Dto/EditTableConfigInput.cs
namespace Fast.Center.Service.Table.Dto;

/// <summary>
/// <see cref="EditTableConfigInput"/> 编辑表格配置输入
/// </summary>
public class EditTableConfigInput
{
    /// <summary>
    /// 表格Id
    /// </summary>
    [LongRequired(ErrorMessage = "表格Id不能为空")]
    public long TableId { get; set; }

    /// <summary>
    /// 表格名称
    /// </summary>
    [StringRequired(ErrorMessage = "表格名称不能为空")]
    public string TableName { get; set; }

    /// <summary>
    /// 备注
    /// </summary>
    public string Remark { get; set; }

    /// <summary>
    /// 更新版本控制字段
    /// </summary>
    [LongRequired(ErrorMessage = "更新版本控制字段不能为空")]
    public long RowVersion { get; set; }
}